db/insert-userData.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO UserData VALUES
+    ((default), 'asd', 'pwd', 'dsa@gmail.com', '123456', '0912-1123', '2022-10-10 11:30:30', 111);
+
+INSERT INTO UserData VALUES
+    ((default), 'asd', 'pwd', 'asd@gmail.com', '123456', '0912-1123', '2022-10-10 11:30:30', 111);
+
+INSERT INTO UserData VALUES
+    ((default), 'asd', 'pwd', 'avg@gmail.com', '123456', '0912-1123', '2022-10-10 11:30:30', 111);
```

Comment:
INSERT INTO no need three times
can the data more diff ?
we will cant see the different of front of user name